src/main/java/uk/nhs/hee/tis/trainee/credentials/api/IssueResource.java

```diff
@@ -89,4 +97,31 @@ ResponseEntity<String> issuePlacementCredential(
       return ResponseEntity.internalServerError().build();
     }
   }
+
+  /**
+   * A callback to log the outcome of the issued resource, and redirect to the redirect_uri.
+   *
+   * @param code             The code returned from the gateway.
+   * @param state            The internal state returned from the gateway.
+   * @param error            The error text, if the credential was not issued.
+   * @param errorDescription The error description, if the credential was not issued.
+   * @param token            The user's authorization token.
+   * @return The response entity redirecting to the issuing redirect_uri.
+   */
+  @GetMapping("/callback")
+  ResponseEntity<String> logIssuedResource(
+      @RequestParam(required = false) String code,
+      @RequestParam(required = false) String state,
+      @RequestParam(required = false) String error,
+      @RequestParam(required = false, value = "error_description") String errorDescription,
+      @RequestHeader(HttpHeaders.AUTHORIZATION) String token) {
```

Comment:
The callback won't include an authorization header, it can be cached on the initial request instead